src/ics/icalendar.py

```diff
@@ -18,6 +18,7 @@ class CalendarAttrs(Component):
     prodid: str = attr.ib(validator=instance_of(str), metadata={"ics_priority": 900})  # default set by Calendar.DEFAULT_PRODID
     scale: Optional[str] = attr.ib(default=None, metadata={"ics_priority": 800})
     method: Optional[str] = attr.ib(default=None, metadata={"ics_priority": 700})
+    name: Optional[str] = attr.ib(default=None, validator=v_optional(instance_of(str)), metadata={"ics_priority": 600})
     # CalendarTimezoneConverter has priority 600
```

Comment:
Please also decrement the priority of `CalendarTimezoneConverter` and update the comment here so that we get a consistent ordering with the name before the timezones.